scss/mixins/_buttons.scss

```diff
@@ -43,21 +43,18 @@
   &:disabled {
     color: $color;
     background-color: $background;
-    border-color: $border;
     // Remove CSS gradients if they're enabled
-    @if $enable-gradients {
-      background-image: none;
-    }
+    background-image: if($enable-gradients, none, null);
```

Comment:
Reason I did this is because the property order will be respected if we use this notation (See inconsistency with the block below)